docs/data/sdks/ampli-overview.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Ampli
+description: Overview to Amplitude Analytics SDK architecture and common usage
+---
+
+## Overview
+
+**Ampli** dynamically generates a light-weight wrapper for the **Amplitude SDK** based on your analytics tracking plan in **Amplitude Data** making event tracking easier and less error prone.
+
+This document provides a high-level overview of Ampli. A [video demo](https://academy.amplitude.com/instrumenting-events-with-amplitude-data-and-the-ampli-cli) is also available in Amplitude Academy.
```

Comment:
Overall looks good and really excited to see our new Ampli page coming out. 

Some possible changes
- Replace this video link with a embedded video so that users are more likely to watch this video (no need to sign up in Academy)
This code snippet is from https://amplitude.wistia.com/medias/4f8ufh6les which is the same source as the Academy video  
```
<script src="https://fast.wistia.com/embed/medias/4f8ufh6les.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_4f8ufh6les videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/4f8ufh6les/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
```
- Deleted images in Amplitude Data because they are all covered in this video and they have bad layouts 
- Replace images in Ampli CLI with code snippets

Reply:
Thanks. 

1. Good call. I added the video at the end of the doc with a link. It is somewhat out of date. I tried putting it at the top but I could find a good way to organize it with the existing diagram and overview section.
2. I want to keep the images so users can see the required steps without seeking through the video. What it bad about the layouts?
3. Thought about this but felt it would be encouraging for the user to see a couple real screenshots. I used snippets for description sections.

Reply:
two concerns 
- images in Amplitude Data. 
  - All three of these thumbnails are very length (they take the whole screen). But they are very small so I cannot see the actual contents of them. 
  - And once I click on one of them, it takes the whole screen so kind of lost in the process of reading 
  - Can we put each of them under each step related? So that at lease I know exactly which step I am on

- images in Ampli CLI
  - There are no explanation around them so hard for readers to know what we want to express and why we put the screenshots here at the first glance. 
    - Can we highlight some lines in the screenshots and add some explanations? 
  - They are pretty small so extra clicks needed. But once I click on one of them it takes the whole screen and I lost my  view to other content. 
     - Would it be possible to make them larger enough so that readers don't need to click?